back/src/Services/MucManager.ts

```diff
@@ -23,69 +25,103 @@ export class MucManager {
         // We initialize the list of variable object at room start. The objects cannot be edited later
         // (otherwise, this would cause a security issue if the scripting API can edit this list of objects)
         if (map) {
-            this.chatZones = MucManager.findChatZonesInMap(map);
-            this.chatZones?.forEach((chatZone) => {
-                chatZone.mucUrl = `${this.roomUrl}/${chatZone?.chatName ?? ""}`;
-                chatZone.mucCreated = false;
+            this.mucRooms = MucManager.findMucRoomsInMap(map);
+            this.mucRooms?.forEach((mucRoom) => {
+                if (mucRoom.type === "forum") {
+                    mucRoom.mucUrl = `${this.roomUrl}/forum/${mucRoom?.chatName ?? ""}`;
+                } else {
+                    mucRoom.mucUrl = `${this.roomUrl}/${mucRoom?.chatName ?? ""}`;
+                }
+                mucRoom.mucCreated = false;
             });
+            // LATER : If we want to let the default forum name to be customisable in a map property ...
+            // if (mapDetails.mucRooms !== null) {
+            //     // If the ADMIN exist we disable all the forums from the MAP
+            //     this.mucRooms.forEach((chat) => {
+            //         if (chat.type === "forum") {
+            //             chat.mucCreated = true;
+            //         }
+            //     });
+            // } else {
+            //     console.log("No admin set");
+            //     // If the ADMIN_URL is not set, so there is no admin, and we define a default chatForum MUC room if it's not defined in the map
+            //     if (this.getDefaultForum() === null) {
+            //         this.mucRooms.set("welcome", {
+            //             chatName: "welcome",
+            //             mucUrl: `${this.roomUrl}/forum/welcome`,
+            //             mucCreated: false,
+            //             type: "forum",
+            //         } as MucRoom);
+            //     }
+            // }
         }
     }
 
     public async init(mapDetails: MapDetailsData) {
-        const allMucRooms = await ejabberdClient.getAllMucRooms();
-        const allMucRoomsOfWorld: string[] = [];
-        if (Axios.isAxiosError(allMucRooms)) {
-            console.warn("Error to get allMucRooms (AxiosError) : ", allMucRooms.response);
-        } else if (allMucRooms instanceof Error) {
-            console.warn("Error to get allMucRooms : ", allMucRooms);
+        const allMucRoomsCreated = await ejabberdClient.getAllMucRooms();
```

Comment:
This will not scale if the system has too many rooms.

Reply:
Maybe we could "cache" the list of live zones and remove the live zones that are not in the cached list every hour.

This way, we make only one request to `getAllMucRooms` per hour?